GitUI/CommandsDialogs/BrowseDialog/GitStatusMonitor.cs

```diff
@@ -335,11 +335,9 @@ private void Update()
                     {
                         try
                         {
-                            await TaskScheduler.Default;
-
                             var cmd = GitCommandHelpers.GetAllChangedFilesCmd(true, UntrackedFilesMode.Default,
                                 noLocks: true);
-                            var output = module.GitExecutable.GetOutput(cmd);
+                            var output = await module.GitExecutable.GetOutputAsync(cmd).ConfigureAwait(false);
```

Comment:
This change causes git-status to run on the UI thread. Is this really necessary?

Reply:
Do you observe any issues related to this? This is an async call and does not block the UI.

Reply:
No problem seem, just confused by it.